### PR TITLE
G20-482: Frontend-changes

### PR DESCRIPTION
**Pull request by mpereiraesaa**
Removed and commented html related to account dropdown, removed code from flag/report feature, added inspera icon to every group, workaround to avoid group name overflow

### CODE REVIEW

File: src/sidebar/templates/annotation.html [hunk 1]
```diff
@@ -141,21 +141,6 @@
           on-close="vm.showShareDialog = false">
         </annotation-share-dialog>
       </span>
-      <span ng-if="vm.canFlag()">
```
Comment (jarlep) [inline src/sidebar/templates/annotation.html]: Wonder if I would have preferred simply commenting out like `<!--`

File: src/sidebar/components/annotation.js [hunk 2]
```diff
@@ -200,29 +200,6 @@ function AnnotationController(
     );
   };
 
-  /**
-    * @ngdoc method
-    * @name annotation.AnnotationController#flag
-    * @description Flag the annotation.
-    */
-  this.flag = function() {
```
Comment (jarlep) [inline src/sidebar/components/annotation.js]: Do we need to remove this method?

File: src/sidebar/templates/annotation.html (hunk 1 above)
Comment (mpereiraesaa) [reply src/sidebar/templates/annotation.html]: sure yeah it is a better option.

File: src/sidebar/components/annotation.js (hunk 2 above)
Comment (mpereiraesaa) [reply src/sidebar/components/annotation.js]: comment this function should be enough.

**Pull request merged**